src/main/java/com/java_essential/Cherednichenko/classwork/lesson_2/Course.java

```diff
@@ -1,20 +1,15 @@
 package com.java_essential.Cherednichenko.classwork.lesson_2;
 
+import java.util.ArrayList;
+
 public class Course {
-    private Student[] students;
+    private ArrayList<Student> students;
```

Comment:
```suggestion
    private ArrayList<Student> students;
```
This is incorrect field declaration
change to `private List<Student> students = new ArrayList<>();`